src/library_html5_webgl.js

```diff
@@ -30,12 +30,12 @@ var LibraryHtml5WebGL = {
       HEAP32[a+i] = 0;
     }
 
-    HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.alpha }}}>>2)] =
```

Comment:
I assume the size of EM_BOOL hasn't changed, was this just wrong before?

Reply:
This change is no longer part of this PR now that #21189 has landed